news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,124 @@
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/re-lm.jpg)](https://osu.ppy.sh/s/588066)
+
+This map caught my eye for a few reasons. The most obvious being the heavy use of SV, which is thoughtfully crafted to be intuitive and smooth, which is quite impressive given how easy it is to cross the line of being more of a distraction than a feature. Though the SV is not the main reason I think this map is particularly deserving of a spotlight. Lude doesn't shy away from LN usage in their mapping of this track. Instead, they use it to add atmospheric value, complexity and originality, which is what I personally love to see. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves, makes this a suitable pick for May's spotlight.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/black-or-white.jpg)](https://osu.ppy.sh/s/565779)
+
+Let me introduce you another extreme 7K marathon map by Kawawa. I really like how the map structure flows nicely with the song intensity. From the map itself, i can see the mapper have a really good experience in term of emphasizing the melody itself. What caught my attention is the jacking style pattern in the map which it is pretty new and interesting to see. Besides giving a great pattern usage, the speed change through the map are astonishing extreme and fun. The mapper really did a good job on hightlighting certain beats magnitude with a suitable slider velocity. By advancing through the map, player could easily experience the consciousness of the music along with the note arrangement emphasized by the mapper intention. Have a try while you are on it.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/shine.jpg)](https://osu.ppy.sh/s/456289)
+
+For his 5th ranked map, Frim4503 offers us another 5K marathon, this time for a drum & bass love live opening remix !
+As usual we have a mix of LN and chords playing really good, making it a really fun map to try big accuracy scores. SV are really lightly used aswell with a quite unique style, focussing on the voice before each kiai. Without changing much of music/mapping style, Frim4503 show us one more time that anti-meta is not dead with a really accessible and catchy map.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/doppelganger.jpg)](https://osu.ppy.sh/s/407153)
+
+Apart from having the hardest difficulty ever ranked in osu!mania, it counts with a really enjoyable and challenging patterns in every single difficulty. We also have a few GDs coming from Evening, jakads and Lude. Evening's chart is not really heavily charted, but watch out the svs! They're way more noticeable in this difficulty than in the other ones. Every single mapper and modder has done a really good job checking the pattern's quality and playability of each difficulty (even though alter ego it's hell itself, someone with a good eye can really see how good the patterns fits with the song).
```

Comment:
Completely agree with all the changes you've made, but I'm not entirely sure about the Evening's chart sentence. I don't really think that really overtly heavy is the best way to set it up since I'm referring to the mapping density and I can't really tell what the comment is talking about with "overtly" in that sentence (It might just be me that I don't get the sentence at all though). Anyways, here's the comment with all the stuff you've told me to fix:

> Apart from having the hardest difficulty ever ranked in osu!mania, it counts with a really enjoyable and challenging patterns in every single difficulty. We also have a few GDs coming from **Evening (https://osu.ppy.sh/u/2193881)**, **jakads (https://osu.ppy.sh/u/259972)** and **Lude (https://osu.ppy.sh/u/6974536)**. **Evening**'s difficulty is not really heavily charted, but watch out the svs! They are way more noticeable in this difficulty than in the other ones. Every single mapper and modder has done a really good job checking the pattern's quality and playability of each difficulty (even though alter ego is probably hell itself, someone with a good eye can really see how good the patterns fits with the song).

I have found a few ways to set the Evening's sentence in a way that everyone could understand it:

- Evening's difficulty is not (really?) heavily mapped/charted
- Evening's difficulty is not as hard as alter ego
- Evening's difficulty is not (really?) densely mapped/charted

I tried to use the most common osu!mania technicisms. All the other changes have been applied by the way.